docs/internals/howto-release-django.txt

```diff
@@ -222,28 +233,28 @@ OK, this is the fun part, where we actually push out a release!
     package and compare them to the checksums listed below.
 
 
-    Release package:
-    ================
+    Release packages:
+    =================
 
     Django <<VERSION>>: https://www.djangoproject.com/m/releases/<<URL>>
 
 
-    MD5 checksum:
-    =============
+    MD5 checksums:
+    ==============
 
     MD5(<<RELEASE TAR.GZ FILENAME>>)= <<MD5SUM>>
```

Comment:
James said he's fine with it. I realized the old format comes from the openssl version of the commands, e.g. `openssl dgst -sha256`.